doc/user/content/integrations/psql.md

```diff
@@ -1,10 +1,14 @@
 ---
-title: "CLI Connections"
+title: "Connect to Materialize via psql CLI"
 description: "You can connect to Materialize from your favorite shell using compatible tools, like mzcli or psql."
+aliases:
+  - /connect/
+  - /connect/cli/
 menu:
   main:
-    parent: "connections"
-    weight: 1
+    parent: "integrations"
+    weight: 5
+    name: "`psql` CLI"
```

Comment:
`psql` looks small in the navbar in Safari but looks good in Chrome.
The same with "`materialized` Binary"

<img width="346" alt="Screen Shot 2022-04-13 at 14 48 34" src="https://user-images.githubusercontent.com/11491779/163240782-c7cec8ad-19ce-4bcf-9fc0-d77454cd83a2.png">

Reply:
good catch!

Reply:
fixed by switching to fira code font 
![image](https://user-images.githubusercontent.com/11527560/163582516-27abb4d8-e6be-44a6-b7cc-454effbe2fa5.png)